craft_providers/base.py

```diff
@@ -544,6 +552,12 @@ def _enable_snapd_service(self, executor: Executor) -> None:
                 executor=executor,
                 timeout=TIMEOUT_SIMPLE,
             )
+            # TODO: Why is Lunar failing to load? For right now just ignore.
```

Comment:
I recall some bugfixes not getting backported.  

As in, we raised a bug report around 22.10-23.04 but it didn't get fixed until 23.10 and CPC didn't backport to older buildd images for interim bases.  I made up those release numbers, but I remember them mentioning this in some of the bug reports.